enterprise/server/util/rbeutil/rbeutil.go

```diff
@@ -20,6 +33,129 @@ const (
 	V2ExecutionIDPrefix = "v2/"
 )
 
+var (
+	defaultPoolName                   = flag.String("remote_execution.default_pool_name", "", "The default executor pool to use if one is not specified.")
+	enableWorkflows                   = flag.Bool("remote_execution.enable_workflows", false, "Whether to enable BuildBuddy workflows.")
+	workflowsPoolName                 = flag.String("remote_execution.workflows_pool_name", "", "The executor pool to use for workflow actions. Defaults to the default executor pool if not specified.")
+	workflowsDefaultImage             = flag.String("remote_execution.workflows_default_image", "", "The default docker image to use for running workflows.")
+	workflowsCIRunnerDebug            = flag.Bool("remote_execution.workflows_ci_runner_debug", false, "Whether to run the CI runner in debug mode.")
+	workflowsCIRunnerBazelCommand     = flag.String("remote_execution.workflows_ci_runner_bazel_command", "", "Bazel command to be used by the CI runner.")
```

Comment:
`remote_execution.*workflow*` flags could go in `enterprise/server/workflow/service.go`